### PR TITLE
Bug 1184020 - [Contacts][NGA] Import curtain.html element when we use gaia-install or reset-gaia

### PR DESCRIPTION
**Pull request by jpruden92**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c9c112bcf30fe4e7dd9d534ff786f3f7ce88662b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7a97d51c70d0730fea26a005533da226b1a4e802

File: shared/pages/import/js/curtain_panel.js [hunk 1]
```diff
@@ -223,11 +226,11 @@ var Curtain = (function() {
...
 
-      form.classList.add('fade-out');
-      form.addEventListener('animationend', function cu_fadeOut(ev) {
-        form.removeEventListener('animationend', cu_fadeOut);
+      panel.classList.add('fade-out');
+      panel.addEventListener('animationend', function cu_fadeOut(ev) {
+        panel.removeEventListener('animationend', cu_fadeOut);
         clean();
-        delete form.dataset.state;
+        delete panel.dataset.state;
```
Comment (borjasalguero) [inline shared/pages/import/js/curtain_panel.js]: `delete form.dataset.state;`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d8949b4529c71a75377c8b100c812da049c83a6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=343a4177d19520ebb05ba9de527f8a31c76edcec

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c6a9e763964a12dedb191740dca9a1543c790882

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2fd5ebb4d0b20a52c0dc74832d2e15b407820ea8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6a42c39c6207dcd1054c160c78c40b4854d0be7c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=603ff40b611ef3b399ba615b048edbf671348525

**Pull request merged**